CMakeLists.txt

```diff
@@ -236,10 +222,65 @@ set(WREN_SRC
 )
 
 add_library(wren STATIC ${WREN_SRC})
-target_include_directories(wren PRIVATE ${THIRDPARTY_DIR}/wren/src/include)
+target_include_directories(wren PUBLIC ${THIRDPARTY_DIR}/wren/src/include)
 target_include_directories(wren PRIVATE ${THIRDPARTY_DIR}/wren/src/optional)
 target_include_directories(wren PRIVATE ${THIRDPARTY_DIR}/wren/src/vm)
 
+################################
+# MRUBY
+################################
+
+find_program(RAKE rake)
+find_program(YACC yacc)
+if(NOT RAKE)
+    message(FATAL_ERROR "Program `rake' not found! Ruby and `rake' are required to build mruby support.")
+endif()
+if(NOT YACC)
+    message(FATAL_ERROR "Program `yacc' not found! Either `yacc' or GNU Bison is required to build mruby support.")
+endif()
```

Comment:
Could these not result in a FATAL_ERROR? Since I don't have `rake` or `yacc` on my machine, I'm no longer able to build TIC-80.

Reply:
Aren't language support modules built unconditionally anyway?

Reply:
In the case of the other scripting languages, the dependencies are packed in. rake or yacc won't be available on a few different build systems. Curious if we could get it to build having needing them in there thinking

Reply:
> rake or yacc won't be available on a few different build systems

``yacc`` is a part of most modern C toolchains. ``rake`` is a build system written in Ruby and usually distributed as a part of a normal Ruby installation. On Windows, all you ever need is the awesome [Ruby Installer](https://rubyinstaller.org/).

Of course, neither of those programs need to be present at runtime.